src/ray/stats/metric.h

```diff
@@ -0,0 +1,192 @@
+#ifndef RAY_STATS_METRIC_H_
```

Comment:
```suggestion
#ifndef RAY_STATS_METRIC_H
```

Remove the trailing `_` in all places.